Update dox

typiconman opened the pull request:
@aleslavista I'm finally taking care of this (sorry). Can you give yourself credit on or around line 1053?

typiconman committed:
Add documentation in Serbo-Croatian

typiconman committed:
Add documentation in Serbo-Croatian (issue #28)

typiconman committed:
Merge branch 'update-dox' of github.com:typiconman/fonts-cu into update-dox

aleslavista commented:
Hello,

I shall do so as soon as I'm done with my book which is to be published in a couple of months. Fact is the documentation needs revision as it was written hastily and moreover I'm not a native speaker, so I'll have a native speaker proofread it.

typiconman committed:
Add Serbian documentation

typiconman committed:
Merge branch 'docwork' into update-dox

typiconman committed:
Fixes to Serbian documentation

typiconman committed:
Remove BOM

aleslavista committed:
Update fonts-churchslavonic-sr.tex

Minor changes to Serbocroatian translation

typiconman committed:
Remove extraneous text

typiconman committed:
Add FiraSlav to sr dox, add sr to Makefile

typiconman commented:
@aleslavista  I added FiraSlav to the Serbian documentation. Can you make sure that it is correct? I did  a simple machine translation.

typiconman committed:
Add sr data to clean recipe of Make

typiconman committed:
Update Serbian documentation (issue #28)

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

typiconman commented:
@aleslavista Are you finished with your updates or will you be doing more work?

aleslavista commented:
> @aleslavista Are you finished with your updates or will you be doing more work?

I"d like to read through the pdf of the last version before handing you the final version, if you don't mind of course.

typiconman committed:
Fix latex syntax

typiconman commented:
Please review the PDF file.
[fonts-churchslavonic-sr.pdf](https://github.com/typiconman/fonts-cu/files/6431277/fonts-churchslavonic-sr.pdf)

aleslavista commented:
> Please review the PDF file.
> [fonts-churchslavonic-sr.pdf](https://github.com/typiconman/fonts-cu/files/6431277/fonts-churchslavonic-sr.pdf)

One think I remarked is that on lines 721 and 732 the double hyphens should be formatted as a dash instead. I could have pasted it from character map but I'm pretty sure this is best handled by LaTeX syntax.

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

Fixed some mistakes

aleslavista committed:
Update fonts-churchslavonic-sr.tex

Fixed more mistakes

typiconman committed:
Fix em-dash

typiconman commented:
> 
> One think I remarked is that on lines 721 and 732 the double hyphens should be formatted as a dash instead. I could have pasted it from character map but I'm pretty sure this is best handled by LaTeX syntax.

I fixed this. It seems that there is a bug in the polyglossia defs for Serbian, so as a workaround, I had to set the language of that phrase to English.

aleslavista commented:
> > One think I remarked is that on lines 721 and 732 the double hyphens should be formatted as a dash instead. I could have pasted it from character map but I'm pretty sure this is best handled by LaTeX syntax.
> 
> I fixed this. It seems that there is a bug in the polyglossia defs for Serbian, so as a workaround, I had to set the language of that phrase to English.

OK, thanks. Do you want me to file a bug report? Also I don't think the current font is adequate for Cyrillic typesetting, we've already discussed it a couple of years ago but in any case the final decision rests on you.

typiconman commented:
I updated the font to Libertinus (rather than the obsolete Linux Libertine). Does it look better?
[fonts-churchslavonic-sr.pdf](https://github.com/typiconman/fonts-cu/files/6441406/fonts-churchslavonic-sr.pdf)

aleslavista committed:
Update fonts-churchslavonic-sr.tex

Fixed more mistakes

aleslavista commented:
> I updated the font to Libertinus (rather than the obsolete Linux Libertine). Does it look better?
> [fonts-churchslavonic-sr.pdf](https://github.com/typiconman/fonts-cu/files/6441406/fonts-churchslavonic-sr.pdf)

Unfortunately no: there are still the same issues with Italic and Bold lingering as there used to be two years ago.

typiconman committed:
Update font

typiconman committed:
Merge branch 'update-dox' of github.com:typiconman/fonts-cu into update-dox

aleslavista committed:
Update fonts-churchslavonic-sr.tex

Fixed date information at the very beginning but I don't know whether LaTeX knows how to handle the Serbocroatian date format.

aleslavista committed:
Update fonts-churchslavonic-sr.tex

Fixed more mistakes

aleslavista committed:
Update fonts-churchslavonic-sr.tex

Fixed more mistakes

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

typiconman commented:
It seems to be OK. Take a look at the attached.
[fonts-churchslavonic-sr.pdf](https://github.com/typiconman/fonts-cu/files/6445208/fonts-churchslavonic-sr.pdf)

You keep saying there are "issues" with the font, but what are the issues? The font developers don't know Serbian typography, so they need specific instructions (e.g., codepoint U+XXXX needs to look like [picture]).

aleslavista commented:
> It seems to be OK. Take a look at the attached.
> [fonts-churchslavonic-sr.pdf](https://github.com/typiconman/fonts-cu/files/6445208/fonts-churchslavonic-sr.pdf)
> 
> You keep saying there are "issues" with the font, but what are the issues? The font developers don't know Serbian typography, so they need specific instructions (e.g., codepoint U+XXXX needs to look like [picture]).

I've already documented everything they needed to know some two years ago. Moreover I do understand these glyphs are optional but consistency is certainly required: now in the upright only some are implemented while italic and bold lack them altogether, and then there's the question of ф looking different in bold from the upright. This font is simply shabby for Cyrillic typesetting, you'll have to concur with me.

typiconman commented:
Maybe I'm not seeing something, but when I look at the PDF, I see the correct form of the letter б (both regular and bold), and the correct forms of п and т in italics. I guess the italic ш is not implemented? But I can't find any use of it in our file.

![sr](https://user-images.githubusercontent.com/5428695/117537171-0250aa80-b008-11eb-8342-abda53434282.png)

aleslavista commented:
> Maybe I'm not seeing something, but when I look at the PDF, I see the correct form of the letter б (both regular and bold), and the correct forms of п and т in italics. I guess the italic ш is not implemented? But I can't find any use of it in our file.
> 
> ![sr](https://user-images.githubusercontent.com/5428695/117537171-0250aa80-b008-11eb-8342-abda53434282.png)

You're right, now that I've seen the new pdf you made this morning I stand corrected. My objections referred to the old version of the font. Never mind Italic ш, it's just optional and seldom seen.

aleslavista commented:
On line 743 I suppose you should write \textrussian before (Выговская пустынь) to mark it as Russian text. Also, I tried making a pdf myself with TeXworks to spare you the trouble of having to constantly make them for me. I did download the fonts but I lack the images and I don't know where to place them so that TeXworks can make use of it.

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

typiconman committed:
Add Russian

typiconman commented:
I updated it.
You can build the PDF with XeTeX, but not with plain TeX. You'll need all of the fonts and the images, which are in the docs/ directory.
On Linux this works automatically by running `make doc`. It's probably quite complicated to set up correctly on Windows.
[fonts-churchslavonic-sr.pdf](https://github.com/typiconman/fonts-cu/files/6446059/fonts-churchslavonic-sr.pdf)

aleslavista committed:
Create fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista commented:
> I updated it.
> You can build the PDF with XeTeX, but not with plain TeX. You'll need all of the fonts and the images, which are in the docs/ directory.
> On Linux this works automatically by running `make doc`. It's probably quite complicated to set up correctly on Windows.
> [fonts-churchslavonic-sr.pdf](https://github.com/typiconman/fonts-cu/files/6446059/fonts-churchslavonic-sr.pdf)

Images must be in the same directory as the source file?

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista commented:
In my opinion, there ought to be more space between the text and the examples after lines 300, 309, 315, 440, 448, 701 and 710.

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista commented:
I still don't understand why on line 312 some words do have the archaic form ZEMLJA and others don't.

aleslavista commented:
I gladly announce you I've finally managed to make a pdf from the sourcefile! So I won't be bothering you anymore! :sweat_smile: Also please notice that on lines 673, 674, 675 the second glyph doesn't look from the same font as the first.

aleslavista commented:
In the text included in tables (i.e. line 763) Cyrillic б is not localized, that is it has the standard look and this clashes with the main body of text. Is it a bug? Do you want me to file a bug for that too? Remember there was also an issue with em-dash.

typiconman commented:
> In the text included in tables (i.e. line 763) Cyrillic б is not localized, that is it has the standard look and this clashes with the main body of text. Is it a bug? Do you want me to file a bug for that too? Remember there was also an issue with em-dash.
> 
> **Update**: I've just noticed this only happens in the pdf I made myself with TeXworks, another flaw is that the numbering of tables doesn't work, so the text reads as follows: "у табели ??"

You have to run XeLaTeX three times to build the pdf.

typiconman commented:
> I gladly announce you I've finally managed to make a pdf from the sourcefile! So I won't be bothering you anymore! sweat_smile Also please notice that on lines 673, 674, 675 the second glyph doesn't look from the same font as the first.

Fixed.

typiconman commented:
> I still don't understand why on line 312 some words do have the archaic form ZEMLJA and others don't.

Only the medial Zemlja should take the archaic form.

typiconman commented:
> In my opinion, there ought to be more space between the text and the examples after lines 300, 309, 315, 440, 448, 701 and 710.

You can try playing around with 
```
\lineskip
\lineskiplimit
```
and `\linespread`.
It turns out more complicated than it seems because of the internal metrics of the fonts as well.

typiconman committed:
Fix typos

aleslavista commented:
> > In the text included in tables (i.e. line 763) Cyrillic б is not localized, that is it has the standard look and this clashes with the main body of text. Is it a bug? Do you want me to file a bug for that too? Remember there was also an issue with em-dash.
> > **Update**: I've just noticed this only happens in the pdf I made myself with TeXworks, another flaw is that the numbering of tables doesn't work, so the text reads as follows: "у табели ??"
> 
> You have to run XeLaTeX three times to build the pdf.

But it takes an insanely long time to make it even just once!

aleslavista commented:
> > In my opinion, there ought to be more space between the text and the examples after lines 300, 309, 315, 440, 448, 701 and 710.
> 
> You can try playing around with
> 
> ```
> \lineskip
> \lineskiplimit
> ```
> 
> and `\linespread`.
> It turns out more complicated than it seems because of the internal metrics of the fonts as well.

Frankly speaking I don't trust myself with such tinkering, I know nothing about LaTeX syntax but it seems to me in other cases spacing was good, that's why the cases I mentioned stand out, they look like out of place with context.

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

Major revision

aleslavista committed:
Update fonts-churchslavonic-sr.tex

Major revision

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

Added missing FiraSlav sample

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

aleslavista committed:
Update fonts-churchslavonic-sr.tex

This ought to be the final version unless I find out further mistakes or misspellings.

aleslavista committed:
Update fonts-churchslavonic-sr.tex

Fixed translation of "Zero-width joiner"

aleslavista committed:
Update fonts-churchslavonic-sr.tex

Final Version ready for release

typiconman committed:
Merge branch 'master' into update-dox
